src/components/DeleteTargetModal/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import KeyboardAreaView from 'components/common/KeyboardAreaView';
+import Modal from 'react-native-modal';
+import { bool, func, object, string } from 'prop-types';
+import strings from 'localization';
+import Button from 'components/common/Button';
+import TopicIcon from 'components/TopicIcon';
+import styles from './styles';
+
+const DeleteTargetModal = ({ isModalVisible, setModalVisible, target, topicIcon }) => {
+  // TO DO: Add deleteTarget action
+  //
+  // const { status } = useStatus(deleteTarget);
+  // const dispatch = useDispatch();
+  // const deleteTargetRequest = useCallback(id => dispatch(deleteTarget(id)), [dispatch]);
+  // const isLoading = status === LOADING;
+  const isLoading = false;
+
+  return (
+    <Modal
+      isVisible={isModalVisible}
+      style={styles.modal}
+      backdropOpacity={0.4}
+      onBackdropPress={!isLoading ? () => setModalVisible(false) : () => null}>
```

Comment:
WDYT about moving this condition inside a function like

`const onBackDropPress = () => !isLoading && setModalVisible(false)`